frontend/sumo-championship/src/styles/Nav.css

```diff
@@ -1,82 +1,87 @@
-.option {
-    position: relative;
-    text-transform: capitalize;
-    display: inline-block;
-}
-
-.nav-inner,
-.options-parent {
+.nav {
     display: flex;
-    align-items: flex-start;
-}
-.options-parent {
-    align-self: stretch;
-    flex-direction: row;
     justify-content: space-between;
-    gap: 20px;
-}
-.nav-inner {
-    width: 158px;
-    flex-direction: column;
-    justify-content: flex-start;
-    padding: 8px 0 0;
-    box-sizing: border-box;
-}
-.login {
-    position: relative;
-    font-size: 16px;
-    text-transform: capitalize;
-    font-family: Inter, serif;
-    color: #000;
-    text-align: left;
-    display: inline-block;
-    min-width: 84px;
-}
-.button,
-.nav {
+    align-items: center;
+    background-color: white;
+  }
+  .options-parent {
+    align-items: center;
     display: flex;
-    flex-direction: row;
-    align-items: flex-start;
-}
-.button {
-    cursor: pointer;
-    border: 0;
-    padding: 8px 30px;
-    background-color: #fff;
-    justify-content: flex-start;
-    white-space: nowrap;
-}
-.button:hover {
-    background-color: #e6e6e6;
-}
-.nav {
+    margin-right: auto;
+  }
+  .button {
+    top: 0;
+    left: 0;
+    border-radius: 10px;
+    background-color: #f37316;
+    padding: 0.6rem;
+    padding-left: 1.2rem;
+    padding-right: 1.2rem;
+    margin: 0 0.5rem;
+    border-radius: 0.5rem;
+    border: none;
+  }
+  .option {
+    display: block;
+    padding: 0.5rem;
+    margin: 0 0.5rem;
+    border-radius: 0.5rem;
+  }
+  .login {
+    color: white;
+  }
+  .button:hover {
+    background-color: #f6a823;
+  }
+  .option:hover {
+    font-size: 1.3rem;
```

Comment:
Remove font scaling on hover, instead of it change color to gray, please. Also please add `transition: 0.3s;` to make animations smooth